CONTRIBUTING.md

```diff
@@ -38,6 +38,26 @@ regular contributor, then you will want to set up
 do that, the CI checks will run locally before you even write your
 commit message.  This speeds up your development cycle considerably.
 
+## Installing Go ##
+
+The quickest way to get started is to install [Nix](https://nixos.org/download.html).
+Not only will this provide you with Go from within the project but an interactive
+development environment. The development environment can be started from within the
+root of the project by running the `nix develop` command.
+
+The [flake.nix](flake.nix) file within the root of this project lists the dependencies
+this project needs as well as the location of the package to build. After the
+installation of Nix, you can build the package by simply running: `nix build`
+
+This will use the Go version specified in the project and if you have a previously
+installed version of Go it will be ignored and untouched.
+
+If you would rather not install Nix you can install Go via
+[brew](https://brew.sh/) by running `brew install golang` You
+also have the option of simply downloading the package from the
+[Go website](https://go.dev/doc/install) and following the detailed
+instructions to add it to your PATH.
```

Comment:
I suggest removing this bit in the interest of trying to force everyone to use Nix.  Nix is vastly preferable to freewheeling, particularly since we don't want to be in the business of helping folks debug their broken development environment.

Reply:
Conversely I'm wary of mandating an ecosystem that has nothing to do with Golang for development. NixOS is its own ecosystem that you can use for Go development (and many other kinds of development) if _you want_.

Reply:
@mcdonnnj the use of Nix in this project is almost comparable to that of virtualenv. That was the intention.

Reply:
Yes but unlike Python Go does not suffer from dependency conflicts. A Go project can list its dependencies and compile a binary without breaking another Go project's configuration. Python, using `pip` etc., cannot do that with any kind of guarantees unless you use virtual environments to isolate package installs. The only reason `viritualenv`/`pyenv` type tools are basically mandatory for Python is because you have to install Python package dependencies into a shared package manager. Go does not suffer from this problem.

Reply:
One benefit of using Nix is that we could control what version of the Go toolchain is being used.  That's more what I was thinking of vice the dependencies.  I foresee potential issues down the line where users' builds fail because they're using an outdated (or too new) version of the Go toolchain.

This is an important point that may require a @cisagov/team-ois discussion to resolve.

Reply:
@mcdonnnj It’s still a nice way to say, “hey here is this file that you run and you will have the ability to build this package with one command.” Think of the big picture. If this becomes part of our development it makes things so easy. Forget about modifying dot files and all of these virtualenv commands.

Reply:
I have no issue with including NixOS directions. My comment was about removing the alternative directions. I do not want to mandate NixOS as the only way to do Golang development.

Reply:
I concede the point.